pallets/settlement-groth16/src/lib.rs

```diff
@@ -0,0 +1,114 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
```

Comment:
Add also

```
#![deny(missing_docs)]
```

Reply:
Added some documentation comments in https://github.com/HorizenLabs/zkVerify/pull/66/commits/0869642be957d026f8a90bb5086d230859d2a69c  :-P